spyder/plugins/application/confpage.py

```diff
@@ -69,7 +69,8 @@ def setup_page(self):
                               'check_updates_on_startup')
 
         # Decide if it's possible to activate or not single instance mode
-        if running_in_mac_app():
+        # ??? Should this apply to all conda-based installers?
+        if is_conda_based_app():
```

Comment:
In that case, perhaps we should do this?
Multiple instances for macOS would have to be investigated later, as you suggest.
```suggestion
        # ??? Should we allow multiple instances for macOS?
        if sys.platform == 'darwin' and is_conda_based_app():
```